### PR TITLE
Fixing php apptype fileupload name

### PR DESCRIPTION
**Pull request by lmanohara**

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 1]
```diff
@@ -873,10 +874,13 @@ var uploadExistingArchive = function(uploadedArtifact, appTypeName, applicationN
     uploadedFileName = uploadedArtifact.getName();
     var fileUploadPath = "/" + TMP_UPLOADED_APPLICATIONS_PATH + "/"
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: use file utils

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 2]
```diff
@@ -873,10 +874,13 @@ var uploadExistingArchive = function(uploadedArtifact, appTypeName, applicationN
...
+
     var uploadApplicationTmpPath;
-    var splitFileExt = uploadedFileName.split(".");
-    if(splitFileExt[1] == "zip"){
-        uploadApplicationTmpPath = fileUploadPath + splitFileExt[0] + ".upload";
+    var checkFileExtZIP = isFileExtZIP(uploadedFileName);
+
+    if(checkFileExtZIP){
+        var fileName = uploadedFileName.substring(0, uploadedFileName.lastIndexOf("."));
+        uploadApplicationTmpPath = fileUploadPath + fileName + ".upload";
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: use comments

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 3]
```diff
@@ -886,6 +890,14 @@ var uploadExistingArchive = function(uploadedArtifact, appTypeName, applicationN
     uploadedArtifact.move(uploadApplicationTmpPath);
 }
 
+var isFileExtZIP = function(uploadedFileName){
+    var fileExt = uploadedFileName.substr(uploadedFileName.lastIndexOf(".") + 1, uploadedFileName.length);
+    if(fileExt == "zip"){
+        return true;
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: use constants

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 4]
```diff
@@ -104,13 +104,14 @@ var addApplication = function (applicationName, applicationRevision, application
...
-            if(splitArtifactExt[1] == "zip"){
-                    uploadArtifactTmpName = splitArtifactExt[0] + ".upload";
-                }else{
-                    uploadArtifactTmpName = uploadedFileName;
-                }
+        var uploadArtifactTmpName;
+        var checkFileExtZIP = isFileExtZIP(uploadedFileName);
+        if (checkFileExtZIP) {
+            var fileName = uploadedFileName.substring(0, uploadedFileName.lastIndexOf("."));
+            uploadArtifactTmpName = fileName + ".upload";
```
Comment (punnadi) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Use a constant for ".upload" and refer it everywhere applicable

Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed in following commit.
https://github.com/wso2/app-cloud/pull/167/commits/023eab2d44e9e77d1b791b6329b69cc00a5c4b76

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 1 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed in following commit.
https://github.com/wso2/app-cloud/pull/167/commits/023eab2d44e9e77d1b791b6329b69cc00a5c4b76

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 2 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed in following commit.
https://github.com/wso2/app-cloud/pull/167/commits/023eab2d44e9e77d1b791b6329b69cc00a5c4b76

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 3 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed in following commit.
https://github.com/wso2/app-cloud/pull/167/commits/023eab2d44e9e77d1b791b6329b69cc00a5c4b76

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 5]
```diff
@@ -886,6 +890,14 @@ var uploadExistingArchive = function(uploadedArtifact, appTypeName, applicationN
     uploadedArtifact.move(uploadApplicationTmpPath);
 }
 
+var isFileExtZIP = function(uploadedFileName){
+    var fileExt = uploadedFileName.substr(uploadedFileName.lastIndexOf(".") + 1, uploadedFileName.length);
+    if(fileExt == "zip"){
```
Comment (Kishanthan) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Formating issues

Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: This is fixed in following pull request.
https://github.com/wso2/app-cloud/pull/235/files

**Pull request merged**